.gitignore

```diff
@@ -0,0 +1,2 @@
+.vs/
+.idea/*
```

Comment:
I use webstorm, so I'm ignoring those files from git so they aren't added to the repo